### PR TITLE
Extend release schedule through v1.9

### PR DESCRIPTION
**Pull request by psschwei**
# Changes

Added a few more releases to the schedule. Made an educated guess on rotating through people based on recency and who was listed as release lead candidates here. 

Apologies for the mass tag, but wanted to make sure all the serving/eventing approvers had a chance to see this.

@yanweiguo @ZhiminXiang @dprotaso @nak3 @matzew @evankanderson @lionelvillard @aliok @pierDipi  @odacremolbap

### CODE REVIEW

**Comment by nak3** [comment]
LGTM

**Review by evankanderson — COMMENTED** [review]
/lgtm

I'll approve after another couple days unless someone objects

File: README.md [hunk 1]
```diff
@@ -62,6 +59,11 @@ This roster is seeded with approvers from the Client workgroups.
 | v1.2    | 2022-01-25   | dprotaso       | psschwei            | 2022-01-18 | 2022-01-26
 | v1.3    | 2022-03-08   | carlisia/kvmware | xtreme-sameer-vohra/kvmware | 2022-03-01 | 2022-03-09
 | v1.4    | 2022-04-19   | nader-ziada    | ikvmw               | 2022-04-12 | 2022-04-20
+| v1.5    | 2022-05-31   | yanweiguo      | lionelvillard       | 2022-05-24 | 2022-06-01
+| v1.6    | 2022-07-12   | ZhiminXiang    | evankanderson       | 2022-07-05 | 2022-07-13
+| v1.7    | 2022-08-23   | dprotaso       | matzew              | 2022-08-16 | 2022-08-24
+| v1.8    | 2022-10-04   | nak3           | lionelvillard       | 2022-09-27 | 2022-10-05
```
Comment (cardil) [inline README.md:65]: Why this doesn't match the dates in release schedule:

https://github.com/knative/community/blob/d12424c57947d7a641621e86532f4e374854aa0e/mechanics/RELEASE-SCHEDULE.md#L25-L26

**Pull request merged**